templates/tfengine/components/resources/healthcare_datasets/main.tf

```diff
@@ -23,6 +23,37 @@ module "{{resourceName . "name"}}" {
 
```

Comment:
Unrelated to this CL, but could you update the version from 1.3.0 to 2.0.0? I just made a release today